exonum-java-binding/service-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -123,12 +124,38 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.22.2</version>
         <configuration>
-          <argLine>
-            -Djava.library.path=${nativeLibPath}
-          </argLine>
+          <excludes>
+            <exclude>**/*IntegrationTest.java</exclude>
+          </excludes>
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>2.22.2</version>
+        <configuration>
+          <includes>
+            <include>**/*IntegrationTest.java</include>
+          </includes>
+          <argLine>-Djava.library.path=${nativeLibPath}</argLine>
+          <systemPropertyVariables>
+            <it.artifactFilename>${artifactFilename}.jar</it.artifactFilename>
+            <it.artifactId>${groupId}:${artifactId}:${version}</it.artifactId>
```

Comment:
Please extract this and https://github.com/exonum/exonum-java-binding/pull/1229/files#diff-77cf287dc69b3029b3d73154618ed696R183 
in a property.